toolkit/xre/nsEmbedFunctions.cpp

```diff
@@ -338,7 +338,17 @@ nsresult XRE_InitChildProcess(int aArgc, char* aArgv[],
   NS_ENSURE_ARG_POINTER(aArgv[0]);
   MOZ_ASSERT(aChildData);
 
+  mozilla::TimeStamp beforeRRInit = mozilla::TimeStamp::Now();
   recordreplay::Initialize(&aArgc, &aArgv);
+  mozilla::TimeStamp afterRRInit = ([]() {
+    mozilla::recordreplay::AutoPassThroughThreadEvents pt;
+    return mozilla::TimeStamp::NowUnfuzzed();
+  })();
+  // Only print init-time message when not replaying.
+  if (!recordreplay::IsReplaying()) {
+    double initTime = (afterRRInit - beforeRRInit).ToMilliseconds();
+    fprintf(stderr, "RecordReplay InitTime: %.2f (ms)\n", initTime);
```

Comment:
Printing things to stderr when recording but not replaying could confuse things without AutoPassThroughThreadEvents.  Instead, this should use recordreplay::PrintLog, which will not interact with the recording and will only be printed with RECORD_REPLAY_VERBOSE.